e2e/process-services-cloud/apps-section-cloud.e2e.ts

```diff
@@ -28,13 +29,33 @@ describe('Applications list', () => {
     const settingsPage = new SettingsPage();
     const appListCloudPage = new AppListCloudPage();
     const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP.name;
+    let identityService: IdentityService;
+    let groupIdentityService: GroupIdentityService;
+    let rolesService: RolesService;
+    let testUser, apsUserRoleId, groupInfo;
+    const apiService = new ApiService(browser.params.config.oauth2.clientId, browser.params.config.bpmHost, browser.params.config.oauth2.host, 'BPM');
 
     beforeAll(async (done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+        identityService = new IdentityService(apiService);
```

Comment:
Can we have a method
CreateUserWithRole that wraps those calls?

Reply:
so you mean, create a method that does the calls to both identity service and roles service? which is a good place to put this?

Reply:
identity service